src/server/generators/titleGenerator.js

```diff
@@ -11,7 +11,7 @@ export default function _titleGenerator (options = {}) {
   return function titleGenerator (type, data) {
     return {
       text () {
-        return `<${type} ${attribute}="true">${data}</${type}>`
+        return data ? `<${type} ${attribute}="true">${data}</${type}>` : ''
```

Comment:
I wouldn't think of this behavior normally (when using vue-meta in context of a typical SPA and not going for microfrontends or similar). Please add at least a comment there :relaxed: 	

Also, maybe checking for `undefined` instead of using all falsy values might make sense? :thinking:

Reply:
I change the checking to `String(data)` for keeping it same behaviour as `document.title=....`

```js
document.title = undefined // <title>undefined</title>
document.title = null // <title>null</title>
document.title = false // <title>false</title>
document.title = 0 // <title>0</title>
document.title = '' // <title> tag will be removed
```

Reply:
It makes sense to be in line with `document.title` here but I'm not sure if being in line with the `vue-meta`-own conventions might be more important (though that will only be relevant for v2)